tests/manage/mcg/conftest.py

```diff
@@ -128,7 +129,7 @@ def awscli_pod(mcg_obj, created_pods):
     Returns:
         pod: A pod running the AWS CLI
     """
-    awscli_pod_obj = helpers.create_pod(namespace='noobaa',
+    awscli_pod_obj = helpers.create_pod(namespace=defaults.ROOK_CLUSTER_NAMESPACE,
```

Comment:
please fix parentheses

Reply:
As stated in previous comments - I recommend accessing `self.namespace` instead

Reply:
ack